packages/contracts/contracts/core/permission/PermissionManager.sol

```diff
@@ -406,6 +426,20 @@ contract PermissionManager is Initializable {
         return keccak256(abi.encodePacked("IMMUTABLE", _where, _permissionId));
     }
 
+    /// @notice Decides if the granting permissionId is restricted when `_who = ANY_ADDR` or `_where = ANY_ADDR`.
+    /// @dev by default, every permission is unrestricted and it's the derived contract's responsibility to override it. NOTE: ROOT_PERMISSION_ID is included and not required to set it again.
+    /// @param _permissionId The permission identifier.
+    /// @return bool Whether ot not permissionId is restricted.
+    function isPermissionRestrictedForAnyAddr(bytes32 _permissionId)
+        internal
+        view
+        virtual
+        returns (bool)
+    {
+        (_permissionId); // silence the warning.
+        return false;
```

Comment:
doing ` (_permissionId); // silence the warning.` consume gas?

```suggestion
    /// @param "bytes32 _permissionId" The permission identifier.
    /// @return bool Whether ot not permissionId is restricted.
    function isPermissionRestrictedForAnyAddr(bytes32)
        internal
        view
        virtual
        returns (bool)
    {
        return false;
```

something like this will remove the warning, i think.

Reply:
Doesn't require any extra gas, so we're good. I like your way as well, but interesting if that works  well with the documentation parser. @Michael-A-Heuer Thoughts ?

Reply:
This will break the NatSpec. Please use parentheses as in the original commit.
I think the compiler is smart enough to realize that this can be optimized away.